README.md

```diff
@@ -77,6 +77,16 @@ To stop the Bucketeer container, when you are done testing, you should run:
 
 You can always see which containers are running by using Docker's `docker ps` command.
 
+## Mocking AWS Lambda's Response
+
+When you run the Bucketeer container on a local machine, there is no AWS Lambda function connected to your local desktop (or laptop). While we hope in the future to orchestrate a fully functional testing environment, where the developer can mock AWS Lambda for testing through the use of [TestContainers](https://www.testcontainers.org/), in the meantime, there is a Bash script that can be run to "mock" the interaction with AWS Lambda. This is only relevant when one is testing batch jobs against Bucketeer. Individual image conversion jobs do not use AWS Lambda and so don't require any mocking of that interaction.
```

Comment:
Nitpicky, but I think "local desktop (or laptop)" could be simplified to "local device".

Reply:
Okay, thanks. Will update.